math/drgeo1/Portfile

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+name                drgeo1
+version             1.1.0
+categories          math
+maintainers         {puffin.lb.shuttle.de:michael.klein @mklein-de} openmaintainer
+license             GPL-2
+platforms           darwin
+description         Dr Geo is an interactive geometry software.
+
+long_description    ${description} This means one can draw geometric figure \
+    based on mathematic property but also can move part of the figure in \
+    respect with its properties. \
+    Dr Geo is an educational oriented software. It's an observatory to \
+    explore geometry situation in an interactive way in opposition to \
+    figure drawn on a sheet of paper. The user interface has been thing to \
+    be simple but efficient for young users. In this idea, Dr Geo can also \
+    handle different languages. It can can be used by mathematic teachers. \
+    In a class room to make a show or in a computer class room. Also the \
+    teacher can share Dr Geo with its students.
+
+homepage            http://drgeo.eu/
+master_sites        sourceforge:project/ofset/drgeo/${version}/
+
+distname            drgeo-${version}
+
+checksums           rmd160  94859c622cf9a48f6134d7e348e6f5e94a43ed6c \
+                    sha256  8c93bb637c11ca1647f90e0d97ea03d5938efddc36c2d73b40f15ff8df672216
+
+depends_lib         port:guile18 \
```

Comment:
You are missing
```
depends_build       port:pkgconfig
```
This shows up if you install the port with `port -t -v install drgeo1`